tunnel.tf

```diff
@@ -41,7 +41,7 @@ resource "google_compute_vpn_tunnel" "tunnel-static" {
 }
 
 resource "google_compute_vpn_tunnel" "tunnel-dynamic" {
-  count         = var.cr_name != "" ? var.tunnel_count : 0
+  count         = var.cr_enabled ? var.tunnel_count : 0
```

Comment:
@nick4fake This logic looks like it will cause a change in behavior. Previously it's the inverse of the static block.

Reply:
Previously, there was reliance on a non-empty name, now boolean variable `cr_enabled` was introduced, and if it is true, the `count` is not zero. And vice versa in the case of a static tunnel. In general, the logic is the same

Reply:
@ingwarr could you please add more info of why have you done this change?

Reply:
@morgante I don't see good workarounds

Reply:
cr_enabled seems as seems an equivalent for cr_name != ""

Reply:
Ok